<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePlayerInfoWidget</class>
 <widget class="QWidget" name="GamePlayerInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;
    border: none;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:none;
     background: transparent;
     background-image:none;
     border:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_playerColor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
      background: transparent;
      background-image:none;
      border:none;
margin-bottom:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resources/Resources.qrc">:/images/player_blue.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>5</y>
      <width>106</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Halo Dek</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(96, 56, 19);
      border-image:none;
      background: transparent;
      background-image:none;
      border:none;
margin-bottom:0;</string>
    </property>
    <property name="text">
     <string>username</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_dice">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>9</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
      background: transparent;
      background-image:none;
      border:none;
margin-bottom:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resources/Resources.qrc">:/images/dice-4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
